adapter/src/main/java/com/twilio/kudu/sql/rules/KuduAggregationLimitRule.java

```diff
@@ -66,9 +67,8 @@
  */
 public class KuduAggregationLimitRule extends RelOptRule {
 
-  private static final RelOptRuleOperand OPERAND = operand(Sort.class,
-      some(operand(Aggregate.class, some(operand(KuduToEnumerableRel.class,
-          some(operand(Project.class, some(operand(Filter.class, some(operand(KuduQuery.class, none())))))))))));
+  private static final RelOptRuleOperand OPERAND = operand(Sort.class, some(operand(Aggregate.class,
```

Comment:
Its not required to have `KuduToEnumerableRel` matched (see `KuduSortRule`) so removed it.

Reply:
Just thinking out loud here - Isn't `executeQuery` method of KuduToEnumerableRel the catch-all kind of relation that gets matched when nothing else matches ? 

If we remove it, all the non matched cases go directly to base calcite code or something else ?

Reply:
Since the pattern still has `KuduQuery` in it will end up calling `executeQuery` eventually. 
Removing the requirement of matching `KuduToEnumerableRel` will allow this rule to get matched if we support some other convention that supports asyn querying in the future. This is just to keep consistent with our exising sort rules
[KuduToEnumerableRel](https://github.com/twilio/calcite-kudu/commit/d68073dc4f4c2ce06f2de43b788d69b57f573612)

Reply:
This is great. One of the things I stumbled on when working on a Calcite and Kudu to JavaRx, this didn't get matched.